src/main/java/net/imagej/table/BoolColumn.java

```diff
@@ -66,31 +68,20 @@ public void setHeader(final String header) {
 		return Boolean.class;
 	}
 
+	// -- PrimitiveColumn methods --
+
 	@Override
-	public void fill(final Boolean[] values) {
-		final boolean[] prim = toPrimitive(values);
-		this.setArray(prim);
+	public void fill(final boolean[] values) {
```

Comment:
Ah, and here I see we are ditching the arrays of wrapped primitives. Great. I guess we talked about this awhile ago, huh?